aiohttp/web_urldispatcher.py

```diff
@@ -145,7 +145,7 @@ class StaticRoute(Route):
 
     def __init__(self, name, prefix, directory, *,
                  expect_handler=None, chunk_size=256*1024,
-                 response_factory=None):
+                 response_factory=None, sendfile_fallback=False):
```

Comment:
Public API is made not for tests, but for users. There is no need to pollute it with special flags, while you may call exact sendfile function in tests.